R/readQFeatures.R

```diff
@@ -11,12 +11,19 @@
 ##'
 ##' - The single-set case will generate a `QFeatures` object with a
 ##'   single `SummarizedExperiment` set containing all features of the
-##'   input table.
+##'   input table. This single-set case should be used when reading 
+##'   data at the peptide or at the protein level. These data are
+##'   contained in a table where the quantitative values for each 
+##'   sample are contained in a separate column. 
```

Comment:
Not sure this last sentence is correct. You can have samples contained in separate columns in the for the multi-set case too.

Reply:
In the multi-set case, the quantitative values of different samples are contained in a shared column. So this sentence is correct (in the way I mean it), but may need to be clarified? Maybe "in a separate column" is not appropriate and "in an individual column" is better?